src/components/dashboards/admin/bars/SideBar.jsx

```diff
@@ -45,6 +46,7 @@ function SideBar({ isOpen }) {
     dispatch(setActiveButton('setting'));
   };
   const handleLogout = () => {
+    localStorage.removeItem('token');
     navigate('/login');
```

Comment:
@Kevin-ishimwe , as our App is not totally pages authenticated , i have opinion that redirecting a user to homepage is better than enforcing him to login after logout!

Reply:
fixed